### PR TITLE
PCUI-1: Added .gitignore

### PR DESCRIPTION
**Pull request by KristinaKots**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,4 @@
 # IntelliJ
-.idea/
```
Comment (ZanderRak) [inline .gitignore:2]: @KristinaKots , why did you remove my changes?

Comment (KristinaKots) [reply .gitignore:2]: There were too much same expressions.

Comment (ZanderRak) [reply .gitignore:2]: Ok. I get it. My fault... 
Thanks.
